src/main/java/com/artipie/auth/AuthFromYaml.java

```diff
@@ -66,7 +59,8 @@ public AuthFromYaml(final YamlMapping cred) {
         Optional<String> res = Optional.empty();
         if (users != null && users.yamlMapping(user) != null) {
             final String stored = users.yamlMapping(user).string("pass");
```

Comment:
@paulodamaso I believe variable name `pass` instead of `stored` would be more appropriate.

Reply:
@Vatavuk I'm already using `pass` for the password value sent from user. I'm using the name `stored` because it represents the value which is stored in our Yaml file.